src/components/WalletDropdown/MiniPortfolio/PortfolioRow.tsx

```diff
@@ -28,39 +26,30 @@ export default function PortfolioRow({
   title,
   descriptor,
   right,
-  setIsHover,
   onClick,
 }: {
   left: React.ReactNode
   title: React.ReactNode
   descriptor?: React.ReactNode
-  right: React.ReactNode
+  right?: React.ReactNode
```

Comment:
if `right` is undefined, maybe the `EndColumn` parent below should also not be rendered?